src/System Application/App/SFTP Client/src/SFTPClient.Codeunit.al

```diff
@@ -0,0 +1,180 @@
+namespace System.SFTPClient;
```

Comment:
All of your files are missing the open source header:

// ------------------------------------------------------------------------------------------------
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License. See License.txt in the project root for license information.
// ------------------------------------------------------------------------------------------------